src/main/resources/templates/schedule/viewschedule.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML>
```

Comment:
This file looks very similar to `index.html`, only with a few changes to the fragments that are used. I like that you guys fragmented out the code, but having this page in addition to `index.html` seems very unnecessary, and could be DRYed up using Thymeleaf if statements.

Reply:
You’re correct. “viewschedule.html” reused all the code from index but would add extra elements. 

However, “index” doesn’t know what scheduleid to display so it would break. I believe this change is a necessary distinction.

Reply:
Can you elaborate on why the "scheduleid" variable needs to be on its own page?

Reply:
"Index" doesn't display the list of courses so it doesn't need a scheduleid.

Reply:
I feel like a better method would be to use a `th-if` to see if a scheduleid exists. if it does, you use `th-if` to display the schedule, otherwise you use `th-unless` to display the index thing.

Reply:
Ohh really good idea. Okay, I understand what you're saying now.